app/meteor-accounts-saml/server/saml_utils.js

```diff
@@ -543,6 +565,20 @@ SAML.prototype.validateResponse = function(samlResponse, relayState, callback) {
 				profile.nameIDFormat = nameID.getAttribute('Format');
 			}
 		}
+
+		const subjectConfirmation = subject.getElementsByTagNameNS('urn:oasis:names:tc:SAML:2.0:assertion', 'SubjectConfirmation')[0];
+		if (subjectConfirmation) {
+			const subjectConfirmationData = subjectConfirmation.getElementsByTagNameNS('urn:oasis:names:tc:SAML:2.0:assertion', 'SubjectConfirmationData')[0];
+			if (subjectConfirmationData) {
+				this.validateNotBeforeNotOnOrAfterAssertions(subjectConfirmationData, callback);
```

Comment:
I don't think `this.validateNotBeforeNotOnOrAfterAssertions` should receive the callback, it should return a `true` or `false` instead, and callback should be called in this scope. It should also `return` in case of not valid as well.